### PR TITLE
plugin: verbosepaste: gives copy-paste verbose progress percentage

### PR DESCRIPTION
**Pull request by wustho**
This may be too simple... Feel free to unmerge...

Description: Simple script to give copy-paste a progress percentage
             by utilizing rsync. This may be so simple,
             but a little bit too complicated to implement as an alias
             or command plugin inside nnn.

             Recommended key bind: NNN_PLUG='p:verbosepaste;'
             So you can paste marked selections by pressing `;p`.

             LIMITATION: this won't work when pasting to MTP device.

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Plugins extend the capabilities of `nnn`. They are _executable_ scripts (or bina
 | [uidgid](uidgid) | List user and group of all files in dir | sh | ls, less |
 | [upgrade](upgrade) | Upgrade nnn manually on Debian 9 Stretch | sh | curl |
 | [upload](upload) | Upload to Firefox Send or ix.io (text) or file.io (bin) | sh | [ffsend](https://github.com/timvisee/ffsend), curl, jq, tr |
+| [verbosepaste](verbosepaste) | Gives copy-paste verbose progress percentage | sh | rsync |
```
Comment (jarun) [inline plugins/README.md:71]: Please use a shorter name... maybe `rsynccp`

File: plugins/verbosepaste [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+# Description: Simple script to give copy-paste a progress percentage
+#              by utilizing rsync. This may be so simple,
+#              but a little bit too complicated to implement as an alias
+#              or command plugin inside nnn.
+#
+#              Recommended key bind: NNN_PLUG='p:verbosepaste;'
```
Comment (jarun) [inline plugins/verbosepaste:8]: Please don't recommend shortcuts or explain common usage in a plugin.

File: plugins/verbosepaste [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# Dependencies: rsync
+#
+# Shell: POSIX compliant
+# Author: Benawi Adha
+
+sel=${NNN_SEL:-${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.selection}
+
+if [ -f "$sel" ]; then
+
+    # Choose one of these two schemes by commenting
```
Comment (jarun) [inline plugins/verbosepaste:22]: Please remove this line. The code is self-explanatory.

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/README.md (hunk 1 above)
Comment (wustho) [reply plugins/README.md]: Well that's better name... Done

**Comment by jarun** [comment]
Thank you!

**Pull request merged**